ROADMAP.md

```diff
@@ -0,0 +1,14 @@
+# BenchPress Roadmap
+
+This feature list is a peek into our roadmap. It identifies some of the significant features we are currently working on and a rough timeframe for when you can expect to see them. It is not comprehensive but is intended to provide some visibility into key investments.
+
+| Feature |
+| --- |
+| Available BenchPress on Powershell Gallery |
+| Feature complete test-generator and make it available on nuget as a tool |
```

Comment:
I would avoid mentioning feature since the spec for test gen has not been fully flushed out. 

```suggestion
| Test Generation is available on  Nuget  |
```